<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerStatCard</class>
 <widget class="QFrame" name="PlayerStatCard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>355</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string>QFrame {
    background-color: white;
    border: 1px solid #e0e0e0;
    border-radius: 10px;
    padding: 20px;
    margin: 5px;
}
QFrame:hover {
    border: 2px solid #4CAF50;
}</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QHBoxLayout" name="mainLayout">
   <property name="spacing">
    <number>30</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="leftLayout" stretch="0,0">
     <property name="spacing">
      <number>8</number>
     </property>
     <item>
      <widget class="QLabel" name="nameLabel">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>球员姓名</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="infoLabel">
       <property name="styleSheet">
        <string notr="true">color: #666; font-size: 16pt;</string>
       </property>
       <property name="text">
        <string>学号: | 性别 | 角色</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="middleLayout" stretch="0,0">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QLabel" name="matchesHeaderLabel">
       <property name="styleSheet">
        <string notr="true">color: #888; font-size: 16pt; font-weight: 500;</string>
       </property>
       <property name="text">
        <string>比赛场次</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="matchesCountLabel">
       <property name="font">
        <font>
         <pointsize>28</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string>color: #333;</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="rightLayout">
     <property name="spacing">
      <number>8</number>
     </property>
     <item>
      <widget class="QLabel" name="winrateHeaderLabel">
       <property name="styleSheet">
        <string notr="true">color: #888; font-size: 16pt; font-weight: 500;</string>
       </property>
       <property name="text">
        <string>胜率</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="winratePercentLabel">
       <property name="font">
        <font>
         <pointsize>28</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string>color: #4CAF50;</string>
       </property>
       <property name="text">
        <string>0.0%</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="winrateProgressBar">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>16</height>
        </size>
       </property>
       <property name="styleSheet">
        <string>QProgressBar {
    border: 1px solid #e0e0e0;
    border-radius: 8px;
    background-color: #f5f5f5;
}
QProgressBar::chunk {
    background-color: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop:0 #4CAF50, stop:1 #66BB6A);
    border-radius: 8px;
}</string>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="recordLabel">
       <property name="styleSheet">
        <string>color: #666; font-size: 12pt; margin-top: 3px;</string>
       </property>
       <property name="text">
        <string>0胜 - 0负</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
